### PR TITLE
UI assorted fixes

### LINKED ISSUE #9368: Copy console UI button does not work
The copy console (copy icon) button in the resource view card on left side of a VM does not work.

##### ISSUE TYPE

 * Bug Report

##### COMPONENT NAME

~~~
UI
~~~

##### CLOUDSTACK VERSION

~~~
4.19/4.19.1 RC1
~~~

##### STEPS TO REPRODUCE

~~~

~~~

##### EXPECTED RESULTS

~~~

~~~

##### ACTUAL RESULTS

~~~

~~~

### PR DESCRIPTION
**Pull request by rohityadavcloud**
UI fixes to address missing useful columns and fixes copy console button.

Remove unnecessary copy button on info card that looks congested and confusing.

Fixes #9368 

### Types of changes

- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] Enhancement (improves an existing feature and functionality)
- [x] Cleanup (Code refactoring and cleanup, that may add test cases)
- [ ] build/CI
- [ ] test (unit or integration test code)

### CODE REVIEW

**Comment by rohityadavcloud** [comment]
@blueorangutan ui

File: ui/src/components/widgets/Console.vue [hunk 1]
```diff
@@ -56,17 +56,10 @@ export default {
         this.url = (json && json.createconsoleendpointresponse) ? json.createconsoleendpointresponse.consoleendpoint.url : '#/exception/404'
         if (json.createconsoleendpointresponse.consoleendpoint.success) {
           if (this.copyUrlToClipboard) {
+            this.$copyText(this.url)
```
Comment (DaanHoogland) [inline ui/src/components/widgets/Console.vue:59]: is this a fix for your safari issue?

**Comment by DaanHoogland** [comment]
looks generally good, will test in FF and Chrome.

**Review by shwstppr — APPROVED** [review]
Code lgtm. May need some manual verifications

**Review by sureshanaparti — APPROVED** [review]
clgtm

**Comment by DaanHoogland** [comment]
works in FF and Chrome, but I don't think we need remove the copy console button here: 
image

**Review by DaanHoogland — APPROVED** [review]

**Review by vishesh92 — APPROVED** [review]
clgtm. Tested changes in the Simulator UI and it's working as expected.

**Review by nvazquez — APPROVED** [review]
Code LGTM

**Comment by rohityadavcloud** [comment]
@DaanHoogland as the copy button isn't what most people want (they could just click the console button), I find that duplicated and  annoying; so only keep the button in the actions button-bar. The info card is supposed for quick/common use-case so console button is still there.

**Pull request merged**